### DISCUSSION TITLE
Kafka REST Proxy authentication

### DISCUSSION THREAD

**Discussion by rruckley** [grafana/grafana#41175] [Alerting]
I work for a Telco and security is a big deal. We want to use Grafana alerting into a Kafka topic via the Kafka REST proxy notifier but it doesn't support any forms of authentication.

I'm no go programmer but I will have an attempt at creating a PR for this, looks straight forward so far. 

Anything else I should be aware of?

**Comment by rruckley** [comment]
Currently the drone build is failing at the lint-frontend step but I haven't modified any front-end code, not sure how to rectify this.

**Comment by rruckley** [comment]
Just waiting on review / approval of the PR now.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
